lib/async/http/body/pipe.rb

```diff
@@ -20,6 +20,9 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 # THE SOFTWARE.
 
+require 'async/io/socket'
+require 'async/io/stream'
```

Comment:
If `Body::Pipe` class is used standalone we get an error unless the above two dependencies are required.